.circleci/config.yml

```diff
@@ -120,13 +120,28 @@ jobs:
       - restore-cache: *restore-bundle-cache
       - run: bundle check --path=vendor/bundle || bundle install --deployment --jobs=4 --retry=3
       - save-cache: *save-bundle-cache
+      - run:
+          name: Setup Environment variables
+          command: |
+              echo 'export ASSET_HOST_URL=$(if [ "$CIRCLE_BRANCH" == "live" ]; then echo "https://browse.goodcity.hk/assets"; else echo "https://browse-staging.goodcity.hk/assets"; fi)' >> $BASH_ENV
+              echo 'export APP_VERSION=$(jq -r ".version" package.json)' >> $BASH_ENV
       - attach_workspace:
           at: ~/code
       - run: mv dist-www/ dist
+      - run:
+          name: Get source map url with grep
+          command: echo 'export SOURCE_MAP_WITH_VARIABLE_NAME=$(grep sourceMappingURL dist/assets/browse-*.js)' >> $BASH_ENV
```

Comment:
Output of this is ** SOURCE_MAP_WITH_VARIABLE_NAME=  //# sourceMappingURL=browse-534b0c957eafe9ba01fceddaea1385e6.map**